File: postman/sdks/Specs SDKs/Search Service/java/src/main/java/com/searchservicesdk/models/ReleasesResponse.java
package com.searchservicesdk.models;

import com.fasterxml.jackson.annotation.JsonIgnore;
import com.fasterxml.jackson.annotation.JsonProperty;
import java.util.List;
import lombok.Builder;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NonNull;
import lombok.ToString;
import lombok.With;
import lombok.extern.jackson.Jacksonized;
import org.openapitools.jackson.nullable.JsonNullable;

/**
 * Response model for the `releases` endpoint.
 */
@Data
@Builder
@With
@ToString
@EqualsAndHashCode
@Jacksonized
public class ReleasesResponse {

  /**
   * List of release models.
   */
  @NonNull
  private List<ReleasesDefaultModel> data;

  @JsonProperty("debug")
  private JsonNullable<Object> debug;

  @JsonProperty("explain")
  private JsonNullable<Object> explain;

  @JsonIgnore
  public Object getDebug() {
    return debug.orElse(null);
  }

  @JsonIgnore
  public Object getExplain() {
    return explain.orElse(null);
  }

  // Overwrite lombok builder methods
  public static class ReleasesResponseBuilder {

    private JsonNullable<Object> debug = JsonNullable.undefined();

    @JsonProperty("debug")
    public ReleasesResponseBuilder debug(Object value) {
      this.debug = JsonNullable.of(value);
      return this;
    }

    private JsonNullable<Object> explain = JsonNullable.undefined();

    @JsonProperty("explain")
    public ReleasesResponseBuilder explain(Object value) {
      this.explain = JsonNullable.of(value);
      return this;
    }
  }
}

File: postman/sdks/Specs SDKs/Search Service/python/src/search_service_sdk/models/releases_response.py
from __future__ import annotations
from typing import List
from pydantic import Field
from typing import Optional
from typing import Union
from .utils.base_model import BaseModel
from .releases_default_model import ReleasesDefaultModel


class ReleasesResponse(BaseModel):
    """Response model for the `releases` endpoint.

    :param debug: debug, defaults to None
    :type debug: dict, optional
    :param explain: explain, defaults to None
    :type explain: dict, optional
    :param data: List of release models.
    :type data: List[ReleasesDefaultModel]
    """

    debug: Optional[dict] = Field(default=None)
    explain: Optional[dict] = Field(default=None)
    data: List[ReleasesDefaultModel] = Field(description="List of release models.")
